exchange/auction.go

```diff
@@ -15,6 +16,29 @@ import (
 	"github.com/prebid/prebid-server/prebid_cache_client"
 )
 
+type DebugLog struct {
+	EnableDebug bool
```

Comment:
There is a duplicate of the Debug name. Consider renaming to `Enabled`.

Reply:
Renamed to `Enabled`